lib/loggerator/metrics.rb

```diff
@@ -1,18 +1,16 @@
-require_relative 'loggerator'
+require "loggerator"
 
 module Loggerator
   module Metrics
     include Loggerator
     extend self
 
-    @@metrics_name = 'loggerator'
-
     def name=(name)
-      @@metrics_name = name
+      Loggerator.config.metrics_app_name = name
     end
```

Comment:
Yeah, lets deprecate them... sounds good. (Probably we're the only user right now, fwiw)